modules/installation-and-upgrade/pages/container-management/_updating-containers.adoc

```diff
@@ -3,7 +3,7 @@
 The {productname} {productnumber} Server container can be updated using the following command:
 
 ----
-mgradm update
+mgradm upgrade
```

Comment:
```suggestion
mgradm upgrade podman
```

Reply:
This is old file so ignore, I didn't want to delete it with Joseph around.